schema/rdl_schema_0.1.json

```
@@ -1025,6 +788,104 @@
 				"Judgement",
 				"Hybrid"
 			]
+		},
+		"Spatial": {
+			"title": "Spatial coverage",
+			"type": "object",
+			"description": "Information about the spatial coverage of the dataset.",
+			"properties": {
+				"countries": {
+					"title": "Countries",
+					"type": "array",
+					"description": "The countries covered by the geographical area, from the open country codelist (ISO 3166-1).",
+					"items": {
+						"type": "string"
+					},
+					"minItems": 1
+				},
+				"gazetteerEntries": {
+					"title": "Gazetteer entries",
+					"type": "array",
+					"description": "Entries from geographical indices or directories describing the geographical area. This field should be used to describe subnational coverage. Use of is recommended.",
+					"items": {
+            "$ref": "#/$defs/Gazetteer_entry"
+          },
+          "uniqueItems": true,
+					"minItems": 1
+				},
+				"bbox": {
+					"title": "Bounding box",
+					"type": "array",
+					"description": "A geographic bounding box delimiting the geographical area.",
+					"items": {
+						"type": "number"
+					},
+					"minItems": 1
+				},
+				"geometry": {
+					"title": "Geometry",
+					"description": "A set of coordinates denoting the vertices of the geographical area.",
+					"$ref": "#/$defs/Geometry"
+				},
+				"centroid": {
+					"title": "Centroid",
+					"type": "array",
+					"description": "The coordinates of the centre of the geographical area.",
+					"items": {
+						"type": "number"
+					},
+					"minItems": 1
+				}
+			}
+		},
+		"Gazetteer_entry": {
+			"title": "Gazetteer entry",
+			"type": "object",
+			"description": "Entry from geographical indices or directories describing the geographical area.",
+			"properties": {
+				"id": {
+					"title": "Identifier",
+					"type": "string",
+					"description": "An identifier drawn from the gazetteer identified in `.scheme`."
+				},
+				"scheme": {
+					"title": "Scheme",
+					"type": "string",
+					"description": "The gazetteer from which the entry is drawn, from the open location_ gazetteers codelist."
```

See comment on `Spatial.countries`.

```suggestion
					"description": "The gazetteer from which the entry is drawn, from the open [location gazetteers codelist](https://rdl-standard.readthedocs.io/en/dev/data_model/codelists.html#location_gazetteers)."
```